VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
match_parent is what got us into the problem in the first place. I saw that
match parent was deprecated, but it is still used a lot.

The bug was caused because the web view window had no idea how big the
container should be before it loaded content.

Does anyone know of a way to accomplish the desired behavior without it?

On Thu, May 5, 2016 at 12:03 PM, Christopher Lee <notifications@github.com>
wrote:

> In VideoLocker/res/layout/fragment_course_combined_info.xml
> <https://github.com/edx/edx-app-android/pull/690#discussion_r62209110>:
>
> > @@ -21,7 +21,7 @@
> >          <RelativeLayout
> >              android:id="@+id/announcement_layout"
> >              android:layout_width="match_parent"
> > -            android:layout_height="wrap_content">
> > +            android:layout_height="fill_parent">
>
> Good catch o.O
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-app-android/pull/690/files/bd33bf34e28604b1e26eff7665bd542a84ec6bea#r62209110>
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX